src/seedu/addressbook/commands/CreateCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.commands;
+
+
+import java.io.*;
+
+/**
+ * Adds a person to the address book.
+ */
+public class CreateCommand extends Command {
+
+    public static final String COMMAND_WORD = "create";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a new user account. "
+            + "Parameters: USERNAME [p]p/PASSWORD \n"
+            + "Example: " + COMMAND_WORD
+            + " user123 p/password123 ";
+
+    public CreateCommand(String username, String password) {
```

Comment:
Maybe can do a User file in the data folder? for an OOP object user(name,password)

Reply:
I think it's a good idea to have a user object with username and password attributes